test.yaml

```diff
@@ -0,0 +1,17 @@
+tests: 
```

Comment:
Also, it would be good to have a function somewhere that checks if the tests for which some params are defined do exist in the test set and bails out // throws an error if the corresponding 'test_whatever.py' does not exist.
This is not urgent for the moment.